ansible/files/telemetry.sh

```diff
@@ -168,11 +172,41 @@ jupyter serverextension enable --py jupyter_spark --user
 jupyter nbextension install --py jupyter_spark --user
 jupyter nbextension enable --py jupyter_spark --user
 
+# Create analyses dir whether we are mounting efs or not
+mkdir -p $HOME/analyses
+
+if [ -n "$EMAIL" ]; then
+    # Mount entire efs locally to check for user's dir 
+    mkdir -p /mnt/efs
+    sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 fs-d0c30f79.efs.us-west-2.amazonaws.com:/ /mnt/efs
+
+    if [ ! -d "/mnt/efs/$EMAIL" ]; then
+        # If no dir for this user, create it, set ownership, add examples
+        sudo mkdir -p "/mnt/efs/$EMAIL"
+
+        sudo chown hadoop:hadoop "/mnt/efs/$EMAIL"
+
+        wget -nc -P "/mnt/efs/$EMAIL/" https://raw.githubusercontent.com/mozilla/emr-bootstrap-spark/master/examples/Telemetry%20Hello%20World.ipynb
```

Comment:
I do agree. But we run into an issue here:
1. We update tutorials, possibly clobbering any changes they've made to the notebooks
2. We update tutorials with -nc, which means they can't get updated, only added if they've been removed
3. We don't update

I'd opt for (3), basically because over time users will think of these as their directories, and having other people muck around with the files in them is never a satisfying experience.